lib/form_data.js

```diff
@@ -354,6 +355,9 @@ FormData.prototype.submit = function(params, cb) {
     , defaults = {method: 'post'}
     ;
 
+  var proxy = process.env.HTTPS_PROXY;
```

Comment:
Ok, it looks like I have got something working without having to modify form-data module:

https://gist.github.com/mably/ffb351f4e4cd0867ee5d86462e266b6f

Could you tell me if it works on your side too?